packages/auth/src/api/account_management/email_and_password.ts

```diff
@@ -69,6 +69,18 @@ export async function updateEmailPassword(
   >(auth, HttpMethod.POST, Endpoint.SET_ACCOUNT_INFO, request);
 }
 
+// Used for linking an email/password account to an existing idToken. Uses the same request/response
+// format as updateEmailPassword.
+export async function linkEmailPassword(
+  auth: Auth,
+  request: UpdateEmailPasswordRequest
+): Promise<UpdateEmailPasswordResponse> {
+  return _performApiRequest<
+    UpdateEmailPasswordRequest,
```

Comment:
Sign up request/response?

Reply:
I was hoping to reuse the Update request/response, but you are right, we need the SignUpRequest and Response to inject recaptcha enterprise fields.

Done.